config/environments/development.rb

```diff
@@ -33,17 +44,23 @@
   config.eager_load = false
 
   require 'rack-webauth/test'
-  config.middleware.use(
-    Rack::Webauth::Test,
-    user: Settings.WEBAUTH.USER,
-    mail: Settings.WEBAUTH.MAIL,
-    ldapprivgroup: Settings.WEBAUTH.LDAPPRIVGROUP,
-    suaffiliation: Settings.WEBAUTH.SUAFFILIATION,
-    displayname: Settings.WEBAUTH.DISPLAYNAME,
-    ldapauthrule: Settings.WEBAUTH.LDAPAUTHRULE
-  )
+  if Settings.WEBAUTH
```

Comment:
ok... i had trouble overriding existing settings with nil.

since laptops are the only instances that set WEBAUTH info, i made it nil by default in the root settings.yml, with a comment about the expected structure (for local settings that override the default).
https://github.com/sul-dlss/argo/pull/58/files#diff-646ad6b10917e4385cbcc8c8524e24a2R88

the private repo with the vanilla laptop configs already has the WEBAUTH settings override needed to fake webauth credentials on laptops.

how's that sound?